packages/react-components/src/dialog/src/DialogTrigger.tsx

```diff
@@ -131,7 +133,7 @@ export function InnerDialogTrigger({
     );
 }
 
-export const DialogTrigger = forwardRef<InnerDialogTriggerProps>((props, ref) => (
+export const DialogTrigger = forwardRef<any, Omit<InnerDialogTriggerProps, "forwardedRef">>((props, ref) => (
```

Comment:
Is there a type similar to ComponentProps but for the a DOM type? Something which would allow us to "DomProps<DefaultElement>"